lib/grammers-mtsender/src/lib.rs

```diff
@@ -540,10 +609,26 @@ pub async fn connect<T: Transport, A: ToSocketAddrs>(
     ))
 }
 
-pub async fn connect_with_auth<T: Transport, A: ToSocketAddrs>(
+pub async fn connect_with_auth<T: Transport>(
     transport: T,
-    addr: A,
+    addr: SocketAddr,
```

Comment:
Why was the generic removed? (Same as before.)

Reply:
Sorry, I'not found way convert ToSocketAddress to ToTargetAddress, I want it to run first